commit a3370a6d6479aecbfd6df18e16f1bbaa0fb1c5ff
Author: Ben Skeggs <bskeggs@redhat.com>
Date:   Fri Mar 17 00:48:29 2017 +0000

    drm/nouveau/disp/nv50-: split chid into chid.ctrl and chid.user
    
    [ Upstream commit 4391d7f5c79a9fe6fa11cf6c160ca7f7bdb49d2a ]
    
    GP102/GP104 make life difficult by redefining the channel indices for
    some registers, but not others.
    
    Signed-off-by: Ben Skeggs <bskeggs@redhat.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1441_a3370a6d6479aecbfd6df18e16f1bbaa0fb1c5ff.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1441_a3370a6d6479aecbfd6df18e16f1bbaa0fb1c5ff.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
